env.example

```diff
@@ -2,4 +2,8 @@ REPORT_A_PROBLEM_ENABLED= # an undefined value means it is not enabled
 NOTIFY_BASE_API_URL=https://api.notification.canada.ca
 NOTIFY_REPORT_A_PROBLEM_EMAIL=experience@servicecanada.gc.ca
 NOTIFY_API_KEY=
-NOTIFY_REPORT_A_PROBLEM_TEMPLATE_ID=
\ No newline at end of file
+NOTIFY_REPORT_A_PROBLEM_TEMPLATE_ID=
+
+NEXTJS_CONTENT_API =  #api address here
+
+NEXTJS_CONTENT_ENABLED = # true means content is taken from AEM api, false means content is taken from local file
```

Comment:
I think these two are pretty much doing the same work, so we can potentially check for the existence of `NEXTJS_CONTENT_API` as our boolean.